cli/azd/cmd/infra_create.go

```diff
@@ -174,14 +174,6 @@ func (i *infraCreateAction) Run(ctx context.Context) (*actions.ActionResult, err
 		}
 	}
 
-	// TO BE MOVED TO DEPLOY OUTPUT
-	// if i.formatter.Kind() != output.JsonFormat {
-	// 	resourceGroupName, err := project.GetResourceGroupName(ctx, i.azCli, prj, env)
-	// 	if err == nil { // Presentation only -- skip print if we failed to resolve the resource group
-	// 		i.displayResourceGroupCreatedMessage(ctx, i.console, env.GetSubscriptionId(), resourceGroupName)
-	// 	}
-	// }
```

Comment:
What does this mean? I'd think logically we'd want the resource group printed out at the end of `infra create`. Putting it in `deploy` doesn't actually make much sense, except for the `up` case (end of the operation).

Reply:
that's the current requirement: https://www.figma.com/proto/iIRRiGPuKIfaSFfMJXcYle/azd-resource-provisioning?node-id=536%3A125681 :)

Reply:
This is also mentioned by @ellismg. I understand the concern and the feedback here.
However, I also understand there was some UX analysis and work performed before to create the requirements and prototypes. 
In an ideal world, we would want to provide as many feedback as possible during the prototype presentation/demos and not during the implementation PR, as the cost is higher at this point.
Since the prototype was showed before and agreed, I don't really want to change it as part o this implementation PR.
If there are changes to be made, I would consider a new iteration to re-visit the requirements, update the prototypes and docs and finally implement any new changes.

Does it make sense?  
I feel like, if we make decisions for how the implementation should work during the PR review, we would be basically ignoring all the previous work from the team to put together the expectations :)

Reply:
@vhvb1989 I think the figma was created on an older version of `azd` that precludes the corrected change to where the resource group output was shown.

> I feel like, if we make decisions for how the implementation should work during the PR review, we would be basically ignoring all the previous work from the team to put together the expectations :)

I'm not sure how the rest of the team feels. My belief is that the figma is merely a sketch of what the design should look like. As we start exploring the actual implementation details in code, we'd need to work iteratively to fine tune the final output UX. Ultimately, the guiding factor should be the end-user experience.

> Since the prototype was showed before and agreed, I don't really want to change it as part of this implementation PR.
If there are changes to be made, I would consider a new iteration to re-visit the requirements, update the prototypes and docs and finally implement any new changes.

Now, we can always question why the requirements were missed, and that is certainly an area of improvement. For example, we could have been more included in the initial design phase to make sure important technical details are captured. However, in the real world, we'll always end up missing something and need to course correct.

@vhvb1989 Is your concern on cost is that the UX language simply doesn't have a story for handling these types of outputs? If so, I still think that is a cost worth looking into as early as possible. We want the UX language to be comprehensive, not limiting.